Exercise_C/index.css

```diff
@@ -2,3 +2,35 @@
 This css sheet is linked to index.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+
+.inlinebox {
```

Comment:
So in this exercise we wanted you to use Flex box to recreate the image you made in the previous exercise. Please come back and attempt to do this.
My advice is to allocate a decent height to your container!